riot-os-posix-lwip/Makefile

```diff
@@ -0,0 +1,89 @@
+APPLICATION = posix-tls
+
+# This has to be the absolute path to the RIOT base directory:
+RIOTBASE ?= $(CURDIR)/../RIOT
```

Comment:
Might be easier out of the box experience using `RIOTBASE ?= $(CURDIR)/../../RIOT`. That allows wolfssl-examples and RIOT to be side by side.

Reply:
Makes sense, fixed. Thanks!